results/cmd/api/main_test.go

```diff
@@ -100,10 +107,9 @@ func TestGetTaskRun(t *testing.T) {
 	if err != nil {
 		t.Fatalf("could not create taskrun: %v", err)
 	}
-	t.Logf("Created taskrun: %s", r.String())
 
 	// Test if we can find inserted taskrun
-	res, err := srv.GetTaskRunResult(ctx, &pb.GetTaskRunRequest{ResultsId: r.GetResultsId()})
+	res, err := srv.GetTaskRunResult(nil, &pb.GetTaskRunRequest{ResultsId: r.GetResultsId()})
```

Comment:
same question as above?

Reply:
Go tests have a timeout flag that can be configured, so we likely don't need to do this unless we have a particular reason to have a shorter per-request timeout.

Reply:
same as above

Reply:
done